etc/helm/pachyderm/templates/console/service.yaml

```diff
@@ -6,6 +6,7 @@ SPDX-License-Identifier: Apache-2.0
 apiVersion: v1
 kind: Service
 metadata:
+  annotations: {{- toYaml .Values.console.service.annotations | nindent 4 }}
```

Comment:
We probably should surround this in an if statement

Reply:
ah, fair, I had seen some places where we let the bare `{}` through but that seems fine

Reply:
It might work, generally safer to go with the if though